views/ticketpool.tmpl

```diff
@@ -17,35 +17,28 @@
-          <input data-target="ticketpool.zoom" data-action="click->ticketpool#onZoom" type="button" class="btn btn_sm all btn-active" value="All Time" name="all">
+          <input data-target="ticketpool.zoom" data-action="click->ticketpool#onZoom" type="button" class="btn btn_sm all btn-active" value="Blocks" name="all">
           <input data-target="ticketpool.zoom" data-action="click->ticketpool#onZoom" type="button" class="btn btn_sm 1d" value="Day" name="day">
           <input data-target="ticketpool.zoom" data-action="click->ticketpool#onZoom" type="button" class="btn btn_sm 1wk" value="Week" name="wk">
           <input data-target="ticketpool.zoom" data-action="click->ticketpool#onZoom" type="button" class="btn btn_sm 1m" value="Month" name="mo">
         </div>
 
         <label>&nbsp;&nbsp;Bars :</label>
         <div class="btn-group" data-toggle="buttons">
-          <input data-target="ticketpool.bars" data-action="click->ticketpool#onBarsChange" type="button" class="btn btn_sm all btn-active" value="All Time" name="all">
+          <input data-target="ticketpool.bars" data-action="click->ticketpool#onBarsChange" type="button" class="btn btn_sm all btn-active" value="Blocks" name="all">
           <input data-target="ticketpool.bars" data-action="click->ticketpool#onBarsChange" type="button" class="btn btn_sm 1d" value="Day" name="day">
           <input data-target="ticketpool.bars" data-action="click->ticketpool#onBarsChange" type="button" class="btn btn_sm 1wk" value="Week" name="wk">
           <input data-target="ticketpool.bars" data-action="click->ticketpool#onBarsChange" type="button" class="btn btn_sm 1m" value="Month" name="mo">
         </div>
 
-        <div id="tickets_by_purchase_date" class="tickets"></div>
-
+        <div id="tickets_by_purchase_date" class="tp-charts"></div>
         <br>
-        <div id="tickets_by_purchase_price" class="tickets"></div>
-
+        <div id="tickets_by_purchase_price" class="tp-charts"></div>
         <br>
-        <div id="tickets_by_type_doughnut" class="doughnutGraph">
-          <canvas id="doughnutGraph" width="220" height="180"
-                  chart-options="{tooltipTemplate: '<%=label%>: <%= numeral(value).format('($00[.]00)') %> - <%= numeral(circumference / 6.283).format('(0[.][00]%)') %>'">
-          </canvas>
-
-          <div class="ticket_by_types">
-            <b style="color:#2971FF;">3 Outputs:</b> - typically solo.<br><br>
-            <b style="color:#FF8C00;">5 Outputs:</b> - typically Voting Service Provider (VSP).<br><br>
-            <b style="color:#41BF53;">More than 5 Outputs:</b> - likely multi-party (split) tickets.<br>
+        <div class="justify-content-between">
+          <div class="dygraph-label dygraph-title mb-1">Ticket Outputs Distribution</div>
+          <div class="row col-lg-12 col-sm-8 d-flex text-center m-auto">
+            <table class="tp-outputs-align" data-target="ticketpool.outputs"></tr></table>
```

Comment:
~~`</tr>`~~. Just noticed this. 

Then you're good.

Reply:
I think all is well. Nothing looks out of place according to me. Right?

Reply:
~~`</tr>`~~

Reply:
Seen.